tracer/build/_build/Build.GitHub.cs

```diff
@@ -45,8 +45,10 @@ partial class Build
     [Parameter("Is the ChangeLog expected to change?", List = false)]
     readonly bool ExpectChangelogUpdate = true;
 
+    [Parameter("Git repository name", Name = "GITHUB_REPOSITORY_NAME")]
+    readonly string GitHubRepositoryName;
```

Comment:
Hmm, I've already set this in pipeline variables.
I need it anyway for `update-github-status.yml` 🤔

Reply:
Sure, but we don't really want to have to explicitly set that locally when it will always (eventually) be dd-trace-dotnet. You could/should also add the following to any target that uses it:

```csharp
.Requires(() => GitHubRepositoryName)
```

Reply:
Oh, yes, make sense, changed.